doc/userguide/configuration/suricata-yaml.rst

```diff
@@ -1975,30 +1975,30 @@ The whole DPDK configuration resides in the `dpdk:` node. This node encapsulates
           copy-iface: none # or PCIe address of the second interface
 
 
-The node `dpdk.eal-params` consists of `DPDK arguments <https://doc.dpdk.org/guides/linux_gsg/linux_eal_parameters.html>`_
-that are usually passed through command line. These arguments are used to initialize and configure EAL.
-Arguments can be specified in either long or short forms. When specifying the arguments, the dashes are omitted.
-Among other settings, this configuration node is able to configure available NICs to Suricata, memory settings or other
-parameters related to EAL.
+The `DPDK arguments <https://doc.dpdk.org/guides/linux_gsg/linux_eal_parameters.html>`_, which are typically provided through the command line, are contained in the node `dpdk.eal-params`. 
```

Comment:
lets wrap the lines for better review reading

Reply:
done :scissors: